earth_enterprise/src/SConstruct

```diff
@@ -218,23 +218,17 @@ is_os_ubuntu = False
 is_min_ubuntu = True
 # /etc/issue was the previous method of checking Ubuntu release version
 # however, some flavors of Ubuntu e.g. MATE, have used varying formats
-# using ilsb_release, subprocess and re is more readable than awk parsing also
-try:
-  vstring=subprocess.check_output('lsb_release -i',shell=True)
-  system=re.split('[\t\.\n]',vstring)
-  if system[1] == "Ubuntu":
-    is_os_ubuntu = True
+# using lsb_release, subprocess and re is more readable than awk parsing also
+if not os.path.isfile('/etc/redhat-release'):
```

Comment:
It looks like we might need to set `is_os_ubuntu` to `True` in here.

Reply:
Actually, it looks like it's not used, so we could just delete it on line 217, but that's fairly minor.